src/app.ts

```diff
@@ -157,6 +157,13 @@ async function downloadPage(title: string, link: string, doToggleMenu: boolean):
 
   await page.addStyleTag({ content: 'div[class^="styles__PrevNextButtonWidgetStyled"], div[class^="styles__Footer"], nav { display: none !important; }' });
 
+  await page.evaluate(() => {
+    const elements = document.getElementsByClassName('Button-ktk5iw-0 CircleButton-f4i43u-0 styles__AnimationPlus-sc-2az9w3-13 dqWxjg')
```

Comment:
Is there any better way to define this class name? I mean as you have done above class^="styles__PrevNextButtonWidgetStyled"

Reply:
That one is hardcoded in their app. But, the one you are using seems like a dynamically generated by react.